### PR TITLE
Quantize elementwise mul

### PR DESCRIPTION
**Pull request by zuzg**
### PR types
New features
### PR changes
 OPs

### Describe
Quantization of elementwise multiplication operator and generalizing functions connected with quantized elementwise operators.

### CODE REVIEW

File: paddle/fluid/framework/ir/graph_pattern_detector.h [hunk 1]
```diff
@@ -1016,20 +1016,20 @@ struct Pool : public PatternBase {
...
-
-  PDNode* operator()(PDNode* x_var, PDNode* y_var);
-
-  PATTERN_DECL_NODE(elementwise_add_op);
-  PATTERN_DECL_NODE(elementwise_add_x);
-  PATTERN_DECL_NODE(elementwise_add_y);
-  PATTERN_DECL_NODE(elementwise_add_out);
+// Elementwise ops
+// Forward pass for element-wise operators (add, mul)
+// elementwise_mul_out is the result of the operator
```
Comment (wozna) [inline paddle/fluid/framework/ir/graph_pattern_detector.h:1021]: I think instead of `elementwise_mul_out ` should be `elementwise_out`.

**Review by wozna — APPROVED** [review]
Only one typo. The rest looks good to me.

**Review by lidanqing-vv — APPROVED** [review]
LGTM

File: paddle/fluid/framework/ir/graph_pattern_detector.h (hunk 1 above)
Comment (lidanqing-vv) [reply paddle/fluid/framework/ir/graph_pattern_detector.h:1021]: Please modify this comment in your next PR. @zuzg Thanks

**Pull request merged**